docs/data/sdks/go.md

```diff
@@ -0,0 +1,404 @@
+---
+title: Go SDK (Alpha)
+description: The Amplitude Go SDK installation and quick start guide.
+icon: fontawesome/brands/golang
+---
+
+!!!info "SDK Resources"
+    - [Go SDK Repository :material-github:](https://github.com/amplitude/Amplitude-Go)
+    - [Go SDK Releases :material-code-tags-check:](https://github.com/amplitude/Amplitude-Go/releases)
+
+--8<-- "includes/no-ampli.md"
+
+The Go SDK lets you send events to Amplitude. This library is open-source, check it out on [GitHub](https://github.com/amplitude/Amplitude-Go).
+
+## Getting Started
+
+### Installation
+
+Install `Amplitude-Go` for analytics:
+
+```bash
+go get https://github.com/amplitude/Amplitude-Go
+```
+
+## Usage
+
+### Initializing SDK
+
+Initialization is necessary before any instrumentation is done. The API key for your Amplitude project is required to construct a Config struct. Use that Config struct to initialize a client struct which implements Client interface. It can be used across requests after it is initialized.
+
+```GO
+
+import "github.com/amplitude/Amplitude-Go/amplitude"
+
+func main() {
+
+    // Create a Config struct 
+	config := amplitude.NewConfig("your-api-key")
+
+	// Pass a Config struct  
+    // to initialize a client struct
+    // which implements Client interface
+	client := amplitude.NewClient(config)
+}
+```
+
+### Configuration
+
+Set your configuration before a client is initialized.
+```Go
+func callback(event Event, code int, message ...string) {
+	//callback function that takes three input parameters
+	//event: the event that triggered this callback
+	//code: status code of request response
+	//message: a optional string message for more detailed information
+}
+
+Config{
+		APIKey:          "your-api-key",
+		FlushInterval:   time.Second * 10,
+		FlushQueueSize:  100,
+		FlushMaxRetries: 5,
```

Comment:
let's remove this in code as well

Reply:
Deleted all fields unused in Alpha version